metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/MemoryPoolSensor.java

```diff
@@ -44,7 +44,8 @@ public void sense(final MetricContext metricContext)
 
     private void reportUsage(MemoryPoolMXBean mxBean, MetricContext metricContext)
     {
-        String name = mxBean.getName();
+        // Names are *all* of the form 'Alpha Beta', e.g. 'Code Cache'.
```

Comment:
Can you change this to speak to why the spaces are being removed, rather than what the input is?
This chunk of code has no insight/can't control the bean names, but it does know spaces aren't valid.